Menu/Menu.jsx

```diff
@@ -17,12 +24,39 @@ class Menu extends MaterialComponent {
       "open-from-bottom-left",
       "open-from-bottom-right"
     ];
+    this._select = this._select.bind(this);
+    this._cancel = this._cancel.bind(this);
   }
   componentDidMount() {
     this.MDComponent = new MDCSimpleMenu(this.control);
+    this.MDComponent.listen("MDCSimpleMenu:selected", this._select);
+    this.MDComponent.listen("MDCSimpleMenu:cancel", this._cancel);
+    toggleMenu(defaultProps, this.props, this.MDComponent);
   }
   componentWillUnmount() {
     this.MDComponent.destroy && this.MDComponent.destroy();
```

Comment:
```destroy``` should probably be called after ```unlisten```. Also I'm not sure if ```unlisten``` is necessary, I'll have a look at the code.

Reply:
fixed it, although unlisten is still there, for cleanup sake